ckan/config/environment.py

```diff
@@ -229,6 +230,11 @@ def update_config():
     # routes.named_routes is a CKAN thing
     config['routes.named_routes'] = routing.named_routes
     config['pylons.app_globals'] = app_globals.app_globals
+    # Pylons to Flask legacy route names mapping0
+    config['legacy_route_mappings'] = json.loads(
```

Comment:
I'm not sure if defining this config option here will allow you to change the config dinamically like in the `change_config` decorator (if the test passes then great)

In any case, user defined routes should add to the existing default ones, so better do:

```python
LEGACY_ROUTE_NAMES = {
    'home': 'home.index',
    'about': 'home.about',
}

def update_config():
    # ...
    if config.get('ckan.legacy_route_mappings'):
        config['legacy_route_mappings'] = LEGACY_ROUTE_NAMES.update(
            json.loads(config['ckan.legacy_route_mappings']))



```

Reply:
i will add this change, it looks much cleaner